docs/pages/workflow/customizing.mdx

```diff
@@ -5,21 +5,19 @@ description: Learn how to add custom native code to your Expo project.
 
 import { Terminal } from '~/ui/components/Snippet';
 
-The Expo Go app is a great tool to get started. It exists to help developers quickly get projects off the ground, experiment with ideas (such as on [Snack](https://snack.expo.dev/)), and share their work with minimal friction. Expo Go makes this possible by including a feature-rich native runtime made up of every module in the [Expo SDK](/versions/latest/), so all you need to do to use a module is install the package and reload your app.
+Just as Expo projects can use third-party React Native packages containing native code, Expo projects are also ready for you to add your own custom native code to your project. Once you create your first [development build](/develop/development-builds/introduction/), you can use any native library or custom native code in your local development environment, as well as in your production app. Development builds work just like Expo Go, allowing you to quickly run your app from the [Expo CLI](/more/expo-cli/), except they include all your specific native dependencies and customizations, and not just what is included with the latest [Expo SDK](/versions/latest/).
```

Comment:
```suggestion
Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a [development build](/workflow/overview/#development-builds) allows you to include your specific native dependencies and customizations and not what's in the latest [Expo SDK](/versions/latest/). These native customizations can be any React Native library or your own custom native code.
```

- Let's prefer using library instead of packages
- Avoid using just
- Readability score was a bit high, not easy to skim so rephrased the text for conciseness
- I feel like we don't need to define what development builds does and instead linked to this: https://docs.expo.dev/workflow/overview/#development-builds
- Do we really need to mention Expo Go here?